server/database-couch/seed.js

```diff
@@ -41,15 +41,15 @@ const seed = async (start, number) => {
       return userData;
     }))
 
-    const response = await user2.bulk({ docs: multipleUsers });
+    const response = await user3.bulk({ docs: multipleUsers });
     console.log('Batch complete!');
     if (number > 0) {
       seedManyUsers(start, number);
     } else {
       console.log('seeding complete')
     }
   };
-  seedManyUsers(0, 10000000);
+  seedManyUsers(11, 1000);
```

Comment:
Why are you inputting 11 and 1000 here and not 0 and 10M?

Reply:
testing :(